docs/releases/2.0.txt

```diff
@@ -54,6 +54,10 @@ Minor features
 * The default iteration count for the PBKDF2 password hasher is increased from
   36,000 to 100,000.
 
+* The :class:`~django.contrib.auth.forms.UserCreationForm` and subclasses will
```

Comment:
I would consider this a bug fix rather than a minor feature so no release note is needed.